### DISCUSSION TITLE
Is it possible to set up a rule based on source IP?

### DISCUSSION THREAD

**Discussion by ItsDrike** [evilsocket/opensnitch#922] [Q&A] [answered]
I'm using opensnitch on my laptop, and I'm often moving between university and home. In university, I'd like to set up some rules that block certain apps, but only while I'm on the university network, as they're not allowed per their policy. 

Ideally, I'd like to do this by somehow being able to specify a source IP (as opposed to just destination IP, which is available in the Network tab from rules). Is something like this possible? I haven't seen an option to set source IP anywhere.

If this isn't supported, any suggestions/workarounds I could use to do this?

**Comment by gustavo-iniguez-goya** [comment, marked as answer]
No yet @ItsDrike, but it can be added, I've also needed it sometimes.

Did you realize if the network interface changes when you are at the university and at home? Because if that's the case maybe you could create rules based on the output network interface.

**Reply by ItsDrike** [reply]
It doesn't sadly. Both use the same wireless interface (wlp4s0). I'd love to see the feature get added then, it would be very useful in these cases, though yeah, it's probably not a common problem.

Should I make a feature request issue for it?

**Reply by gustavo-iniguez-goya** [reply]
yes, open a feature request please. I'm adding it and the changes are minimal. I'll test it alittle bit more.

**Reply by ItsDrike** [reply]
Created #924
